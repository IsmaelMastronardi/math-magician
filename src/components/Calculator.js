function CalcItem(props) {
  const item = props;
  return (
    <button type="button" className="symbol">
      {item}
    </button>
  );
}
function CalcItem0(props) {
  const item = props;
  return (
    <button type="button" className="symbol0">
      {item}
    </button>
  );
}

function ResultBar() {
  return (
    <article className="calculator">
      <input className="resultBar" placeholder="0" />
      <div className="calculatorItems">
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem('10')}
        {CalcItem0('0')}
        {CalcItem('8')}
        {CalcItem('8')}
      </div>
    </article>
  );
}

export default ResultBar;
