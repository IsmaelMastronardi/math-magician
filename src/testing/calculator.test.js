import { render } from '@testing-library/react';
import Calculator from '../components/Calculator';

describe('Renders a calculator', () => {
  test('render calculator', () => {
    const { calculator } = render(<Calculator />);
    expect(calculator).toMatchSnapshot();
  });
});
