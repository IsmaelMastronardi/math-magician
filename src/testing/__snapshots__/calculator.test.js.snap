// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a calculator render calculator 1`] = `undefined`;

exports[`Renders a home page render calculator 1`] = `undefined`;
